test_utils/__init__.py

```diff
@@ -160,6 +159,17 @@ def assert_url(first, second):
 
     assert first == second
 
+def assert_url_contains_query_parameters(url, query_params):
+    """
+    Asserts that a url string contains the given query parameters
+    :param url: Full url string to check
+    :param query_params: Dict of query string key/value pairs
+    :raises: Assertion error if the params do not exist in the given url
```

Comment:
We try to use [Google style docstrings](https://google.github.io/styleguide/pyguide.html#Comments).